.circleci/main.yml

```diff
@@ -801,14 +801,14 @@ jobs:
           path: test-results
 
   jupyter-extension-integration-test:
-    resource_class: xlarge
+    resource_class: large
```

Comment:
you mean to do this?

Reply:
uhh I did, but I realize this probably isn't the PR to do this. I'll revert it